src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeFetchers.scala

```diff
@@ -502,27 +502,27 @@ trait BlocksDataFetchers {
   }
 
   /** a fetcher of baker rolls for blocks */
-  implicit val bakersFetcher = new FutureFetcher {
+  implicit val bakersFetcherByHash = new FutureFetcher {
     import JsonDecoders.Circe.Votes._
     import cats.instances.future._
 
     type Encoded = String
-    type In = Block
+    type In = String
     type Out = List[Voting.BakerRolls]
 
-    private val makeUrl = (block: Block) => s"blocks/${block.data.hash.value}/votes/listings"
+    private val makeUrl = (blockHash: String) => s"blocks/$blockHash/votes/listings"
 
     override val fetchData =
       Kleisli(
         blocks => {
-          logger.info("Fetching bakers in levels {}", blocks.head.data.header.level to blocks.last.data.header.level)
+          logger.info("Fetching {} bakers", blocks.size)
```

Comment:
This is misleading, no? It says bakers but it actually displays the number of blocks.

Reply:
changed